suites/pacific/cephadm/test_5-2_new-feature.yaml

```diff
@@ -0,0 +1,138 @@
+      abort-on-fail: true
+  - test:
+      name: Cephadm Bootstrap with apply-spec option.
+      desc: bootstrap with apply-spec option.
+      module: test_bootstrap.py
+      config:
+        command: bootstrap
+        base_cmd_args:
+          verbose: true
+        args:
+          registry-json: registry.redhat.io
+          custom_image: true
+          mon-ip: node1
+          fsid: f64f341c-655d-11eb-8778-fa163e914bcc
+          orphan-initial-daemons: true
+          ssh-user: cephuser
+          ssh-public-key: /home/cephuser/.ssh/id_rsa.pub # if ssh-public-key is provided then provide
+          ssh-private-key: /home/cephuser/.ssh/id_rsa # ssh-private-key also else validation fails
+          apply-spec:
+            - service_type: host
+              address: true
+              labels:
+                - admin
+                - mgr
+                - alertmanager
+              nodes:
+                - node1
+                - node2
+                - node3
+            - service_type: mon
+              placement:
+                nodes:
+                  - node1
+                  - node2
+                  - node3
+            - service_type: mgr
+              placement:
+                label: mgr
+            - service_type: prometheus
+              placement:
+                count: 1
+                nodes:
+                  - node1
+            - service_type: grafana
+              placement:
+                nodes:
+                  - node1
+            - service_type: alertmanager
+              placement:
+                count: 2
+                label: alertmanager
+            - service_type: node-exporter
+              placement:
+                host_pattern: "*"
+            - service_type: crash
+              placement:
+                host_pattern: "*"
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
+      name: Cephadm support for raw mode OSDs using CLI
+      desc: Add raw osd on node3
+      module: test_daemon.py
+      polarion-id: CEPH-83574990
+      config:
+        command: add
+        service: osd
+        pos_args:
+          - "node3"
+          - "/dev/vdb"
+        args:
+          method: raw
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
+      name: Run IOs on the cluster
+      module: test_parallel.py
+      parallel:
+        - test:
+            config:
+              script-name: test_Mbuckets_with_Nobjects.py
+              config-file-name: test_Mbuckets_with_Nobjects.yaml
+              timeout: 300
+            desc: test to create "M" no of buckets and "N" no of objects
+            module: sanity_rgw.py
+            name: Test M buckets with N objects
+        - test:
+            name: rbd-io
+            module: rbd_faster_exports.py
+            config:
+              io-total: 100M
+            desc: Perform export during read/write,resizing,flattening,lock operations
+      desc: Running i/o's parallelly
+  - test:
```

Comment:
do we need to keep old test case here? can we make sure it would be executed in regression stages.